app/src/main/java/de/westnordost/streetcomplete/about/ChangelogFragment.kt

```diff
@@ -76,14 +76,15 @@ private fun readChangelog(resources: Resources) =
 
 private fun addedLinks(description: String): String {
     return description
-        .replace(Regex("([\\s,(])#([0-9]*)")) { matchResult ->
+        .replace(Regex("([\\s,(])#([0-9]+)")) { matchResult ->
```

Comment:
Actually by the same argument I think `,` is redundant in the before match here too.

We can go one better in fact and use a positive lookbehind to not even need the s bit:
`(?<=[\s(]|^)#(\d+)`

Your site can't visualise it, but this one can:
https://www.debuggex.com/